### ISSUE TITLE
Can I use the Macbook to do the DiaNN?

### ISSUE THREAD

**Issue by lzy604** [vdemichev/DiaNN#428]
Dear authors, 

It is really a nice software. I want to use my MacBook to do the analysis in terminal. Is there any codes for this? Thanks

**Comment by vdemichev** [comment]
No, unfortunately does not work on Mac

**Comment by nuan604** [comment]
Is there any methods to install this software on the sever to perform the analysis? I mean not by using the interface one, just use the command in terminal/

**Comment by vdemichev** [comment]
Yes, the Linux version.

**Comment by njohnso6** [comment]
Unfortunately, it is difficult to install on a server using an rpm package since as users we don't have installation permissions. Is there any way other than with Visual Studio to build it in order to just use the command line as a standalone program, without the GUI? I don't see any other makefiles.

**Comment by helloimbavani** [comment]
Hi, 

Is it possible to run DIA-NN on a virtual computer? I tried exactly that, but all i got was the following:

diann.exe --f "C:\Proteomics\Lysate_BG\230424_BG_OE_C20_pooled_5.raw
" --f "C:\Proteomics\Lysate_BG\230424_BG_OE_HMC3_pooled_7.raw
" --lib "C:\Proteomics\Lysate_BG\Output_FragPipe\library.tsv" --threads 4 --verbose 1 --out "C:\Proteomics\Lysate_BG\Output_diaNN\report.tsv" --qvalue 0.01 --matrices  --out-lib "C:\Proteomics\Lysate_BG\Output_diaNN\report-lib.tsv" --gen-spec-lib --predictor --reannotate --fasta "C:\Proteomics\uniprot-download_true_format_fasta_query__28_28proteome_3AUP00000564-2023.06.07-05.44.51.46.fasta" --fasta-search --min-fr-mz 200 --max-fr-mz 1800 --met-excision --cut K*,R* --missed-cleavages 1 --min-pep-len 7 --max-pep-len 30 --min-pr-mz 300 --max-pr-mz 1800 --min-pr-charge 1 --max-pr-charge 4 --unimod4 --var-mods 1 --var-mod UniMod:35,15.994915,M --var-mod UniMod:1,42.010565,*n --monitor-mod UniMod:1 --reanalyse --relaxed-prot-inf --smart-profiling --peak-center --no-ifs-removal 

DIA-NN exited
DIA-NN-plotter.exe "C:\Proteomics\Lysate_BG\Output_diaNN\report.stats.tsv" "C:\Proteomics\Lysate_BG\Output_diaNN\report.tsv" "C:\Proteomics\Lysate_BG\Output_diaNN\report.pdf"
PDF report will be generated in the background


Any thoughts, help or suggestions would be appreciated.

**Comment by vdemichev** [comment]
What do you see if you launch diann.exe from the command line? Was it installed in a regular way using the installer?

**Comment by helloimbavani** [comment]
Yes, I did install it the normal way. I just downloaded it off github (https://github.com/vdemichev/DiaNN/releases/tag/1.8.1)
